ocs_ci/helpers/dr_helpers.py

```
@@ -3375,3 +3391,60 @@ def extract_images_from_yaml(obj, images=None):
+        TimeoutExpiredError: If the cluster does not become unreachable within timeout.
+
+    """
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    mc_obj = OCP(kind=constants.ACM_MANAGEDCLUSTER)
+    logger.info(
+        "Waiting for managed cluster '%s' to become unreachable on ACM hub",
+        cluster_name,
+    )
+
+    def _is_unreachable():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_is_unreachable`

Add return type annotation: `bool`

(ANN202)